Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -132,7 +132,7 @@ export class dotNetExe {
         }
         if (failedProjects.length > 0) {
             if (this.command === 'publish' && !MessagePrinted) {
-                tl.warning(tl.loc('NetCore3Update'));
+                tl.warning(tl.loc('NetCore5Update'));
```

Comment:
I guess this one is no longer required. We had added this as we knew of breaking change in case of publish and wanted to  bubble up that more explicitly as a warning.

I don't think this applies anymore with .Net 5. 
We can also remove that variable `MessagePrinted` then.